internal/domain/indexer.go

```diff
@@ -339,6 +339,7 @@ type TorrentBasic struct {
 	TorrentId string `json:"TorrentId,omitempty"`
 	InfoHash  string `json:"InfoHash"`
 	Size      string `json:"Size"`
+	Uploader  string `json:"Uploader"`
```

Comment:
is this relevant to any other API members?

Reply:
Will be for RED, OPS to start with. Maybe PTP later!